src/encoded/vis_indexer.py

```diff
@@ -245,12 +248,27 @@ def all_visualizable_uuids(registry):
 class VisIndexer(Indexer):
     def __init__(self, registry):
         super(VisIndexer, self).__init__(registry)
+        self.es = registry[ELASTIC_SEARCH]
+        self.esstorage = registry[STORAGE]
+        self.index = registry.settings['snovault.elasticsearch.index']
         self.state = VisIndexerState(self.es, self.index)  # WARNING, race condition is avoided because there is only one worker
 
     def get_from_es(request, comp_id):
         '''Returns composite json blob from elastic-search, or None if not found.'''
         return None
 
+    def update_objects(self, request, uuids, xmin):
```

Comment:
Update objects was move to region and vis indexer child classes because snovault indexer class has a different update objects now.